### PR TITLE
Refactor geometry input

### PR DESCRIPTION
**Pull request by dalg24**

### CODE REVIEW

**Comment by dalg24** [comment]
addressing Issue #38

File: cpp/source/equivalent_circuit.cc [hunk 1]
```diff
@@ -25,13 +25,12 @@ void compute_equivalent_circuit(
   if (!output_database->empty())
     throw std::runtime_error("output_database was not empty...");
 
-  double const sandwich_height =
-      input_database->get<double>("geometry.sandwich_height");
-  double const cross_sectional_area = sandwich_height * 1.0;
+  double const cross_sectional_area =
+      0.0001 * input_database->get<double>("geometry.geometric_area");
```
Comment (Rombur) [inline cpp/source/equivalent_circuit.cc]: Change 0.0001 to a constant so we remember where it comes from. The same below.

File: cpp/source/equivalent_circuit.cc [hunk 2]
```diff
@@ -25,13 +25,12 @@ void compute_equivalent_circuit(
...
   // clang-format off
-  double const electrode_width = input_database->get<double>("geometry.anode_electrode_width");
-  double const separator_width = input_database->get<double>("geometry.separator_width");
-  double const collector_width = input_database->get<double>("geometry.anode_collector_width");
+  double const electrode_width = 0.01 * input_database->get<double>("geometry.anode_electrode_thickness");
+  double const separator_width = 0.01 * input_database->get<double>("geometry.separator_thickness");
+  double const collector_width = 0.01 * input_database->get<double>("geometry.anode_collector_thickness");
```
Comment (Rombur) [inline cpp/source/equivalent_circuit.cc]: Are you sure that you want the names in the variables to be different that the names in the code :s

File: cpp/test/data/verification_problems.info [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+verification_problem_srinivasan {
+    terms_in_truncation_of_infinite_series 10000
+
+    initial_voltage       2.5    ; [volts]
+    discharge_current     5.0e-3 ; [amperes]
+    discharge_time        2.0    ; [seconds]
+    time_step             0.01   ; [seconds]
+
+    percent_tolerance     0.1    ; [percent]
```
Comment (Rombur) [inline cpp/test/data/verification_problems.info:23]: Is this correct that the tolerance is 0.001?

**Comment by dalg24** [comment]
BOOST_CHECK_CLOSE uses percent tolerance.
0.1% here is equivalent to a 0.0001 relative tolerance.

File: cpp/source/equivalent_circuit.cc (hunk 2 above)
Comment (dalg24) [inline cpp/source/equivalent_circuit.cc]: not worth the trouble if new geometries are going to be implemented in the future.

File: cpp/test/data/verification_problems.info (hunk 3 above)
Comment (dalg24) [inline cpp/test/data/verification_problems.info:23]: Correct.
BOOST_CHECK_CLOSE uses percent tolerance.
0.1% here is equivalent to a 0.001 relative tolerance.

File: cpp/source/geometry.templates.h [hunk 4]
```diff
@@ -36,13 +36,13 @@ std::tuple<std::pair<dealii::Point<2>, dealii::Point<2>>,
...
-  double const cathode_collector_width = database->get<double>("cathode_collector_width");
-  double const sandwich_height         = database->get<double>("sandwich_height");
-  double const tab_height              = database->get<double>("tab_height");
+  double const anode_collector_width   = 0.01 * database->get<double>("anode_collector_thickness");
+  double const anode_electrode_width   = 0.01 * database->get<double>("anode_electrode_thickness");
+  double const separator_width         = 0.01 * database->get<double>("separator_thickness");
+  double const cathode_electrode_width = 0.01 * database->get<double>("cathode_electrode_thickness");
+  double const cathode_collector_width = 0.01 * database->get<double>("cathode_collector_thickness");
+  double const sandwich_height         = 0.0001 * database->get<double>("geometric_area");
+  double const tab_height              = 0.01 * database->get<double>("tab_height");
```
Comment (Rombur) [inline cpp/source/geometry.templates.h]: Can you do the same here. Also I think that we should move the definition of `to_meters` to another file. So if one day we need to change to feet, there is only place in the code to change.

File: cpp/source/geometry.templates.h [hunk 5]
```diff
@@ -102,14 +102,14 @@ std::tuple<std::pair<dealii::Point<3>, dealii::Point<3>>,
...
-  double const sandwich_depth          = database->get<double>("sandwich_depth");
-  double const tab_height              = database->get<double>("tab_height");
+  double const anode_collector_width   = 0.01 * database->get<double>("anode_collector_thickness");
+  double const anode_electrode_width   = 0.01 * database->get<double>("anode_electrode_thickness");
+  double const separator_width         = 0.01 * database->get<double>("separator_thickness");
+  double const cathode_electrode_width = 0.01 * database->get<double>("cathode_electrode_thickness");
+  double const cathode_collector_width = 0.01 * database->get<double>("cathode_collector_thickness");
+  double const sandwich_height         = 0.01 * sqrt(database->get<double>("geometric_area"));
+  double const sandwich_depth          = sandwich_height;
+  double const tab_height              = 0.01 * database->get<double>("tab_height");
```
Comment (Rombur) [inline cpp/source/geometry.templates.h]: same here

Comment (dalg24) [inline cpp/source/geometry.templates.h]: aww man

**Pull request merged**